MMUL-BASH-01/Exercises/Day-2/Exercise.006.solutions

```diff
@@ -0,0 +1,45 @@
+1) Supposing your user is "mmul" and your machine is "machine":
+
+> ssh mmul@machine
+
+2) Touch the file in order to create it
+
+touch ~/service_action
+
+3) Using our favourite editor, edit the file properly
+   (rember to save your work!)
+
+vim ~/service_action
+
+4) Touch the file and then edit the content as requested
+
+touch $(whoami)_exercise.service
+vim mmul_exercise.service
+
+5) Using the command that you already know, 'chmod'
+
+chmod 644 mmul_exercise.service
```

Comment:
Please change using $(whoami) instead of mmul

Reply:
replaced username with $(whoami) command